independent-publisher-connectors/HubSpot CRM/apiDefinition.swagger.json

```diff
@@ -32,588 +91,895 @@
+        "operationId": "ContactsUpdate",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ContactID"
+          },
+          {
+            "$ref": "#/parameters/ContactEntry"
+          },
+          {
+            "$ref": "#/parameters/IDProperty"
+          }
+        ]
+      }
+    },
+    "/crm/v3/objects/contacts/search": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "total": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "total"
+                },
+                "results": {
+                  "type": "array",
+                  "description": "array of results of type 'Contact Record'",
+                  "items": {
+                    "$ref": "#/definitions/ContactRecord"
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Input error, e.g. exceeds page limit or too many characters",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          },
+          "401": {
+            "description": "Unauthorized"
+          },
+          "403": {
+            "description": "Forbidden"
+          },
+          "429": {
+            "description": "Too many requests"
+          },
+          "502": {
+            "description": "Timeout"
+          },
+          "503": {
+            "description": "Service temporarily unavailable"
+          },
+          "504": {
+            "description": "Timeout"
+          },
+          "521": {
+            "description": "Web server is down"
+          },
+          "522": {
+            "description": "Connection timed out"
+          },
+          "523": {
+            "description": "Origin is unreachable"
+          },
+          "524": {
+            "description": "Timeout"
+          },
+          "525": {
+            "description": "SSL issues"
+          },
+          "526": {
+            "description": "SSL issues"
+          },
+          "default": {
+            "description": "Unexpected Error"
+          }
+        },
+        "summary": "Search for contacts",
+        "description": "Search for a contacts with the given filters and return a list.",
+        "operationId": "ContactsSearch",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Search"
+          }
+        ]
+      }
+    },
+    "/crm/v3/objects/deals": {
+      "get": {
         "responses": {
           "default": {
             "description": "The default response.",
             "schema": {}

```

Comment:
@ryanedshannon schema is not allowed within the default response. please remove it.  Please fix all default responses in the swagger file.

Reply:
@Amjed-Ayoub I believe that's all responses accounted for, let me know if there are any other issues. Thank you for the help.